pyrate/config.py

```diff
@@ -466,12 +468,62 @@ def original_ifg_paths(ifglist_path):
     :return: full path to ifg files
     :rtype: list
     """
-
     basedir = os.path.dirname(ifglist_path)
     ifglist = parse_namelist(ifglist_path)
     return [os.path.join(basedir, p) for p in ifglist]
 
+def coherence_path_for(path, params, tif=False) -> str:
+    """
+    Returns path to coherence file in tif format for given interferogram.
+
+    Args:
+        path: Path to intergerogram to find coherence file for.
+        params: Parameter dictionary.
+        tif: Find converted tif if True, else find .cc file.
+
+    Returns:
+        Path to coherence file in tif format.
+    """
+    _, file = split(path)
+    pattern = re.compile(r'\d{8}-\d{8}')
+    epoch = re.match(pattern, file).group(0)
+    coh_dir = params.get(COH_DIR)
+    coh_dir = params[OBS_DIR] if coh_dir is None else coh_dir
+    ext = '_cc.tif' if tif else '.cc'
+    coh_path = glob2.glob(os.path.join(coh_dir, '**', f'{epoch}*{ext}'))
+    if len(coh_path) == 0:
+        raise IOError(f"No coherence files found for ifg with epoch " 
+                      f"{epoch}. Check that the correct coherence files "
+                      f"exist in {coh_dir}.")
+    elif len(coh_path) > 1:
+        raise IOError(f"Found more than one coherence file for ifg with epoch "
+                      f"{epoch}. Check that the correct coherence files "
+                      f"exist in {coh_dir}. Found:\n {coh_path}")
+    else:
+        return coh_path[0]
```

Comment:
Can you add an example of expected coherence file name in comments.

Reply:
And done